test/Integration/Dialect/TOSA/bfloat16_cast.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: mlir-opt %s --pass-pipeline="builtin.module(func.func(tosa-to-linalg-named, tosa-to-linalg))" | mlir-opt --linalg-fuse-elementwise-ops --eliminate-empty-tensors --empty-tensor-to-alloc-tensor --one-shot-bufferize="allow-return-allocs allow-unknown-ops bufferize-function-boundaries function-boundary-type-conversion=identity-layout-map" --drop-equivalent-buffer-results --buffer-results-to-out-params --buffer-deallocation --canonicalize --cse --convert-linalg-to-affine-loops | FileCheck %s
```

Comment:
tosa lowering pipeline was refactored to https://github.com/Xilinx/mlir-aie/blob/main/test/Integration/lit.local.cfg. Example can be found at https://github.com/Xilinx/mlir-aie/blob/main/test/Integration/Dialect/TOSA/bf16xbf16_sub_elem/bf16xbf16_sub_elem.mlir. Please update the lowering script, thank you!